docs/network/evm-nodes.md

```diff
@@ -214,11 +214,10 @@ If you want a mode with a longer retention period, you can:
 - Create your own snapshot with the appropriate retention period from another EVM node.
 - Start with a node in a mode that has the necessary data and [switch to another mode](#switching-history-modes) with the retention period that you want.
 
-To download and import the snapshot manually, download the appropriate snapshot for the network and mode (such as from http://snapshotter-sandbox.nomadic-labs.eu/) and import it with the `octez-evm-node snapshot import` command, as in this example:
+To download and import the snapshot manually, download the appropriate snapshot for the network and mode (such as from http://snapshotter-sandbox.nomadic-labs.eu/) and import it with the `octez-evm-node snapshot import` command, as in this example, which uses `<SNAPSHOT_FILE_NAME>` for the snapshot file:
```

Comment:
You can give an URL to the octez-evm-node directly as well (and it will require less disk because you won’t have a temporary file)

Reply:
That's covered in the [following lines](https://github.com/etherlinkcom/docs/blob/3884ebc1962dde99ee8333ba27ff6262b02e669f/docs/network/evm-nodes.md?plain=1#L224), unless I misunderstand something. Maybe I should reverse the order because it's easier to pass the URL to the command.